react/utils/index.tsx

```diff
@@ -21,3 +21,63 @@ export interface PaymentGroupInfo {
   url: string | null
   value: number
 }
+
+export const getSubscriptionInfo = (
```

Comment:
this looks more like two different functions to me

Reply:
Created new function `isSubscription` that checks if a given `Attachment` is a subscription.